cats/cats.py

```diff
@@ -271,9 +285,12 @@ def cli(
     else:
         final_bundle_dump = json.dumps(final_bundle.to_json_dict(), sort_keys=True, indent=4)
 
-    confirmation = input(
-        "The transaction has been created, would you like to push it to the network? (Y/N)"
-    ) in ["y", "Y", "yes", "Yes"]
+    confirmation = pushtx
+
+    if not quiet:
+        confirmation = input(
+            "The transaction has been created, would you like to push it to the network? (Y/N)"
+        ) in ["y", "Y", "yes", "Yes"]
```

Comment:
What happens if quiet is true and push_tx is false?

Reply:
If quiet is true and push is false then the program will exit without pushing the transaction and without prompting. This is how it had previously worked. It seemed slightly convoluted but I wanted to support no prompt for both automatically pushing or not pushing the transaction. If you can think of a cleaner way to do this I would certainly be open to it.